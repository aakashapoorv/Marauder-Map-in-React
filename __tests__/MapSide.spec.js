import React from "react";
import MapSide from "../src/MapSide";
import renderer from "react-test-renderer";
import Enzyme, { shallow, render, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

global.shallow = shallow;
global.render = render;
global.mount = mount;

describe("MapBox tests", () => {
  let MapBoxElement;

  beforeEach(() => {
    MapBoxElement = renderer.create(<MapSide side={1} map={8} isback={true} />);
  });

  it("Should Render correctly", () => {
    let tree = MapBoxElement.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
