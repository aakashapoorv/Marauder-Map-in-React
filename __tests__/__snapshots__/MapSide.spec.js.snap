// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapBox tests Should Render correctly 1`] = `
<div
  className="map-side side-1"
>
  <div
    className="front map8"
  />
  <div
    className="back"
  />
</div>
`;
